### ISSUE TITLE
Close button missing

### ISSUE THREAD

**Issue by seghier** [d3ssy/GhComponentConfigurator#5]
Thank you this is very useful,
Can you add an option to assign icon , and a close button ?

**Comment by d3ssy** [comment]
Yep! Will do! are you finding the tool useful?

**Comment by seghier** [comment]
Yes of course, it save time , i suggest to share it in Rhino forum
